path_config.go

```diff
@@ -159,16 +159,36 @@ func (b *kubeAuthBackend) pathConfigWrite(ctx context.Context, req *logical.Requ
 		return logical.ErrorResponse("no host provided"), nil
 	}
 
-	disableLocalJWT := data.Get("disable_local_ca_jwt").(bool)
+	disableLocalCAJwt := data.Get("disable_local_ca_jwt").(bool)
 	pemList := data.Get("pem_keys").([]string)
 	caCert := data.Get("kubernetes_ca_cert").(string)
 	issuer := data.Get("issuer").(string)
 	disableIssValidation := data.Get("disable_iss_validation").(bool)
 	tokenReviewer := data.Get("token_reviewer_jwt").(string)
 	useAnnotationsAsAliasMetadata := data.Get("use_annotations_as_alias_metadata").(bool)
 
-	if disableLocalJWT && caCert == "" {
-		return logical.ErrorResponse("kubernetes_ca_cert must be given when disable_local_ca_jwt is true"), nil
+	// hasCerts returns true if caCert contains at least one valid certificate. It
+	// does not check if any of the certificates from caCert are CAs, although that
+	// might be something that we want in the future.
+	hasCerts := func(certBundle string) bool {
+		var b *pem.Block
+		rest := []byte(certBundle)
+		for {
+			b, rest = pem.Decode(rest)
+			if b == nil {
+				break
+			}
+
+			if pem.EncodeToMemory(b) != nil {
+				return true
+			}
+		}
+
+		return false
+	}
+
+	if caCert != "" && !hasCerts(caCert) {
+		return logical.ErrorResponse("The provided CA PEM data contains no valid certificates"), nil
```

Comment:
nit: the line is bit long, may bring the message down one line, as it was before.